wire-ios/Wire-iOS Tests/BaseSnapshotTestCase.swift

```diff
@@ -16,13 +16,16 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import XCTest
+import SnapshotTesting
 import WireCommonComponents
+import XCTest
 
 class BaseSnapshotTestCase: XCTestCase {
 
     override func setUp() {
         super.setUp()
+        // Enable when the design of the view has changed in order to update the reference snapshots
+        isRecording = strcmp(getenv("RECORDING_SNAPSHOTS"), "YES") == 0
```

Comment:
> With this change every time the design of a view changes we will update reference snapshot images automatically.

How does this update the snapshots automatically? You still need to set the `RECORDING_SNAPSHOTS` environment var when you want to record snapshots.

Reply:
If you already have screenshots, you get the errors that the screenshots are not matching. When you delete those screenshots and run the tests it automatically record new ones.

Reply:
Will new and old screenshots be shown in the PR?

Reply:
Yes, that won't change.

Reply:
Are you explicitly setting this environment var when you want to record new tests?